editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -1029,6 +978,44 @@ impl Fsm for PathToolFsmState {
 				}
 
 				match self {
+					Self::Ready => {
```

Comment:
Prefer using `if self == Self::Ready` instead of a match since there is only one match arm.